src/status_im/ui/screens/events.cljs

```diff
@@ -201,20 +201,16 @@
    (fx/merge cofx
              {:db (assoc db :view-id view-id)}
              #(case view-id
-                :chat (chat.loading/load-messages cofx)
-                :home (chat.loading/offload-all-messages cofx)
-                :keycard-settings (hardwallet/settings-screen-did-load %)
-                :reset-card (hardwallet/reset-card-screen-did-load %)
-                :enter-pin-login (hardwallet/enter-pin-screen-did-load %)
-                :enter-pin-sign (hardwallet/enter-pin-screen-did-load %)
-                :enter-pin-settings (hardwallet/enter-pin-screen-did-load %)
-                :enter-pin-modal (hardwallet/enter-pin-screen-did-load %)
-                :keycard-login-pin (hardwallet/enter-pin-screen-did-load %)
-                :add-new-account-pin (hardwallet/enter-pin-screen-did-load %)
-                :hardwallet-connect (hardwallet/hardwallet-connect-screen-did-load %)
-                :hardwallet-connect-sign (hardwallet/hardwallet-connect-screen-did-load %)
-                :hardwallet-connect-settings (hardwallet/hardwallet-connect-screen-did-load %)
-                :hardwallet-connect-modal (hardwallet/hardwallet-connect-screen-did-load %)
+                :chat                             (chat.loading/load-messages cofx)
+                :home                             (chat.loading/offload-all-messages cofx)
+                :keycard-settings                 (hardwallet/settings-screen-did-load %)
```

Comment:
Remove all on-will-focus events for keycard

Reply:
Mostly they are used to clean up stored in re-frame pin code. To avoid the leak of that pin in other pin screens better to not store it, and pass as an argument to events which will use it.